dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResourceHelper.java

```diff
@@ -162,8 +162,14 @@ public void saveContent(final String pageId,
                         @Override
                         public void run() {
                             try {
-                                final Contentlet contentlet =
+                                Contentlet contentlet =
                                         contentletAPI.findContentletByIdentifierAnyLanguage(contentletId, variantName);
```

Comment:
I think it is ok, but I am wondering if you should not encapsulate this into a method such as 

findContentletByIdentifierAnyLanguageAnyVariant